main.cpp

```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cat
+//  action 1:    the cat meows
+cat.meow();
+//  action 2:    the cat leaps
+cat.leap();
+//  action 3:    the cat plays jazz
+cat.playJazz();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        door
+//  action 1:    the door opens
+door.open();
+//  action 2:    the door closes
+door.close();
+//  action 3:    the door slams
+door.slam();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        flower
+//  action 1:    the flower grows
+flower.grow();
+//  action 2:    the flower blooms
+flower.bloom();
+//  action 3:    the flower withers
+flower.wither();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        hand
+//  action 1:    the hand clenches
+hand.clench();
+//  action 2:    the hand opens
+hand.open();
+//  action 3:    the hand claps
+hand.clap();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        phone
+//  action 1:    the phone rings
+phone.ring();
+//  action 2:    the phone catches fire
+phone.catchFire();
+//  action 3:    the phone connect to WiFi
+phone.connectToWifi();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        wind
+//  action 1:    the wind blows
+wind.blow();
+//  action 2:    the wind howls
+wind.howls();
+//  action 3:    the wind propels a craft
+wind.propel();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        computer
+//  action 1:    the computer computes
+computer.compute();
+//  action 2:    the computer crashes
+computer.crash();
+//  action 3:    the computer connects to internet 
+computer.connectToInternet();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        ball
+//  action 1:    the ball bounces
+ball.bounce();
+//  action 2:    the ball rolls
+ball.roll();
+//  action 3:    the ball breaks a window
+ball.breakWindow();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        robot
+//  action 1:    the robot builds
+robot.build();
+//  action 2:    the robot cleans
+robot.clean();
+//  action 3:    the robot subjugates humanity
+robot.createTheMatrix();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.